Makefile

```diff
@@ -48,4 +41,4 @@ install: compile
 include $(CURDIR)/build/ci.mk
 include $(CURDIR)/build/release.mk
 
-.PHONY: all build clean validate compile test integration-test install
\ No newline at end of file
+.PHONY: all build clean validate compile test integration-test install
```

Comment:
Shouldn't we remove validate?

Reply:
Absolutely, thanks!  Deleted: [fbe9aa5](https://github.com/newrelic/nri-apache/pull/76/commits/fbe9aa539fb57009e99bbf26e34fb2478de9af03)